src/h5cpp/node/dataset.hpp

```diff
@@ -566,7 +567,7 @@ class DLL_EXPORT Dataset : public Node
     filter::ExternalFilters filters() const;
 
   private:
-    datatype::Datatype file_type;
+    datatype::Datatype file_type_;
```

Comment:
why does the variable name has to be changed?

Reply:
I change the class variable because it is the same as a write method parameter and clang report it as a warning.